test/resolver.content.js

```diff
@@ -226,16 +226,16 @@ tap.test('resolver.content() - throwable:false - remote can not be resolved - "o
     const to = destinationBufferStream(result => {
         t.equal(result, '');
         t.ok(outgoing.success);
+        t.end();
     });
 
     outgoing.pipe(to);
 
     const content = new Content();
-    await content.resolve(outgoing);
-    t.end();
+    content.resolve(outgoing);
 });
```

Comment:
After the change, Tap was unhappy with mixing promises and callbacks so I rewrote the test to be callback only and everything was happy. Not sure why this was working ok before or what exactly made it change but it does seem more correct to me now.

Reply:
:+1: